<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>SelectFishery</class>
 <widget class="QDialog" name="SelectFishery" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Select Fishery</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="fishery_text" >
        <property name="text" >
         <string>Select Fishery :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fishery_comboBox" >
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Shark and Skates</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Coastal reef fish</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Deep reef fish</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Migratory fish</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Benthic invertebrate</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Shrimp</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Other</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbnFisheryFinished" >
        <property name="text" >
         <string>Exit Fishery Interview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnGearFinished" >
        <property name="text" >
         <string>Switch Vessel / Gear Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnStartShapes" >
        <property name="text" >
         <string>Draw Shapes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
